lib/public_key/src/pubkey_cert.erl

```diff
@@ -2195,3 +2196,20 @@ otp_cert(#'OTPCertificate'{} = Cert) ->
     Cert;
 otp_cert(#cert{otp = OtpCert}) ->
     OtpCert.
+
+key_params(#'OTPCertificate'{tbsCertificate = TBSCert}, Params) when Params == asn1_NOVALUE;
+                                                                     Params == 'NULL' ->
+    #'OTPTBSCertificate'{signature =
+                             #'SignatureAlgorithm'{algorithm = Algo,
+                                                   parameters = KeyParams}
+                        } = TBSCert,
+    %% Sometimes parameters may be missing in issuer's "SubjectPublicKeyInfo" but included in
+    %% the certs "SignatureAlgorithm" for RSA PSS signatures.
+    case Algo of
```

Comment:
maybe this case is not needed?

- if you match on `?'id-RSASSA-PSS'` in function head
- 2nd case clause would be handled by existing 2nd function clause

Reply:
I did not due avoiding long lines, but I split it differently so as to be able to match in function head which I normally prefer.